src/murmur/murmur_ice/murmur_ice.pro

```diff
@@ -11,7 +11,12 @@ slice.output = ${QMAKE_FILE_BASE}.cpp
 win32 {
 	slice.commands = slice2cpp --checksum -I\"$$ICE_PATH/slice\" ${QMAKE_FILE_NAME}
 } else {
-	slice.commands = slice2cpp --checksum -I/usr/local/share/Ice -I/usr/share/Ice/slice -I/usr/share/slice -I/usr/share/Ice-3.4.1/slice/ -I/usr/share/Ice-3.3.1/slice/ -I/usr/share/Ice-3.4.2/slice/ -I/usr/share/Ice-3.5.0/slice/ -I/usr/share/Ice-3.5.1/slice/ ${QMAKE_FILE_NAME}
+	ICE_VERSION = $$system(slice2cpp --version 2>&1)
+	EXTRA_SLICEFLAGS =
+	!isEmpty(ICE_VERSION) {
```

Comment:
So `--version` was introduced with 3.6?
I think we should document that on the `ICE_VERSION` variable.